.travis.yml

```diff
@@ -1,19 +0,0 @@
-dist: trusty
```

Comment:
A bit sneaky to remove TravisCI in the same commit :p

Though indeed it seems to not have run since 2 years:
https://app.travis-ci.com/github/ffi/ffi/builds
So I think OK to remove.

Reply:
> A bit sneaky to remove TravisCI in the same commit :p

Lol. It was just confusing when updating the list of rubies as it wasn't clear wether it was still in use or not, hence the delete :)

Thanks for the merge!